felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -778,11 +782,14 @@ func (m *bpfEndpointManager) updateHostIP(ip net.IP, ipFamily int) {
 		// Should be safe without the lock since there shouldn't be any active background threads
 		// but taking it now makes us robust to refactoring.
 		m.ifacesLock.Lock()
+		defer m.ifacesLock.Unlock()
```

Comment:
the point of no doing defer was not to host the lock across the rest of the function. But it may not matter since this is probably not called in parallel with anything, is it?

Reply:
I will better remove the defer and put the unlock, back where it was.